### ISSUE TITLE
can not work with the last ProtocolLib

### ISSUE THREAD

**Issue by wanyanjiabin** [MyPetORG/MyPet#1567]
Mypet version: B1700

when i use the ProtocolLib#593 (last version). The server crashes

```
[16:22:01] [Paper Watchdog Thread/ERROR]: The server has not responded for 10 seconds! Creating thread dump
[16:22:01] [Paper Watchdog Thread/ERROR]: ------------------------------
[16:22:01] [Paper Watchdog Thread/ERROR]: Server thread dump (Look for plugins here before reporting to Paper!):
[16:22:01] [Paper Watchdog Thread/ERROR]: ------------------------------
[16:22:01] [Paper Watchdog Thread/ERROR]: Current Thread: Server thread
[16:22:01] [Paper Watchdog Thread/ERROR]: 	PID: 24 | Suspended: false | Native: false | State: WAITING
[16:22:01] [Paper Watchdog Thread/ERROR]: 	Stack:
[16:22:01] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.3/java.lang.Object.wait(Native Method)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftFuture.get(CraftFuture.java:53)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftFuture.get(CraftFuture.java:40)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[宠物]MyPet-3.12-SNAPSHOT-B1700.jar//de.Keyle.MyPet.util.hooks.ProtocolLibHook$2.onPacketReceiving(ProtocolLibHook.java:152)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.SortedPacketListenerList.invokeReceivingListener(SortedPacketListenerList.java:114)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketRecieving(SortedPacketListenerList.java:67)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.PacketFilterManager.postPacketToListeners(PacketFilterManager.java:549)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.PacketFilterManager.invokePacketReceiving(PacketFilterManager.java:514)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.netty.manager.NetworkManagerInjector.onPacketReceiving(NetworkManagerInjector.java:117)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.processInboundPacket(NettyChannelInjector.java:516)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.lambda$processInboundPacket$2(NettyChannelInjector.java:500)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		[前置]ProtocolLib_dev593.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelInjector$$Lambda$13149/0x0000000803610440.run(Unknown Source)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1491)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:446)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1415)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1188)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:303)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		net.minecraft.server.MinecraftServer$$Lambda$4366/0x00000008014d6e78.run(Unknown Source)
[16:22:01] [Paper Watchdog Thread/ERROR]: 		java.base@17.0.3/java.lang.Thread.run(Thread.java:833)
[16:22:01] [Paper Watchdog Thread/ERROR]: ------------------------------
[16:22:01] [Paper Watchdog Thread/ERROR]: --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH ---
[16:22:01] [Paper Watchdog Thread/ERROR]: ------------------------------
[16:22:06] [Paper Watchdog Thread/ERROR]: --- DO NOT REPORT THIS TO PAPER - THIS IS NOT A BUG OR A CRASH  - git-Paper-81 (MC: 1.19) ---
```

I contacted the developers of ProtocolLib, but they said they would need you to modify part of the code to fix it

https://github.com/dmulloy2/ProtocolLib/issues/1826

**Comment by mercurialmusic** [comment]
FWIW, I've had other plugins warn about the latest ProtocolLib builds being super buggy, many recommending to downgrade to b589 until they get things worked out.

**Comment by Jakllp** [comment]
Should be fixed in `3.12-SNAPSHOT-B1701`
